torch/_torch_docs.py

```diff
@@ -1901,6 +1901,12 @@ def merge_dicts(*dicts):
     y_i = \min(\max(x_i, \text{min\_value}), \text{max\_value})
 """ + r"""
 
+.. note::
+    If :attr:`min` is greater than :attr:`max` :func:`torch.clamp(..., min, max) <torch.clamp>`
+    sets all values equal to :attr:`max`.
+
+    This is shown in the second example below.
```

Comment:
We can probably get rid of this sentence

Reply:
The reference to the example is deleted with the example itself.